setup.py

```diff
@@ -9,7 +9,7 @@
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
     url="https://github.com/centrefornetzero/smart-building-rating-calculator",
-    packages=find_packages("src", package_dir={"": "src"}),
+    package_dir={"": "src"},
```

Comment:
Why are you removing `package_dir={"": "src"}`?

Reply:
Replacing `packages=find_packages()` with `package_dir={"": "src"}` as a bug fix (error message stated that find_packages has no `package_dir` key word)

Reply:
>Replacing packages=find_packages() with package_dir={"": "src"} as a bug fix (error message stated that find_packages has no package_dir key word)

I think that's cause it's supposed to be in a separate argument!